config/helm/ec2-metadata-test-proxy/values.yaml

```diff
@@ -4,6 +4,8 @@ ec2MetadataTestProxy:
     interruptionNoticeDelay: 15
     port: 1338
     label: ec2-metadata-test-proxy
+    enableScheduledMaintenanceEvents: false
```

Comment:
meh, it's just the test proxy :) might as well default this to true, since it's not like we're worried about backwards compat or anything.